app/assets/stylesheets/success_stories.css.scss

```diff
@@ -0,0 +1,25 @@
+.success-stories-photo {
+  border-radius: 50%;
+  height: 120px;
+  width: 120px;
+}
+
+.success-stories-photo--left {
+  margin-left: 15px;
+}
+
+.success-stories-photo--right {
```

Comment:
@rickr I went ahead and just tore out those modifier classes (--left & --right) - no need to have them once I added in the `nth-of-type` CSS rules you suggested.